src/style/profile.less

```diff
@@ -0,0 +1,121 @@
+
+#logoface {
+    max-width: 100px;
+    margin-top: 50px;
+    margin-bottom: 10px;
+    justify-content: center
+}
+
+.ProfilName{
+    background-color: #007EA7;
+    color: #f9fdff;
+    margin-top: 10px;

```

Comment:
you can use the short version margin : top right bottom left (ex margin : 10px, auto, 80px, auto)